### ISSUE TITLE
Hi, would you upload a jupyter notebook about how to use your code?

### ISSUE THREAD

**Issue by tbwxmu** [aksub99/MolDQN-pytorch#1]

**Comment by aksub99** [comment]
Hi,
The code is still under development. I will definitely add a jupyter notebook tutorial once it is complete.
Thanks.

**Comment by EXJUSTICE** [comment]
Tagging on this question, is there a basic example case we can run? Readme is sparse, so not sure  how to go about it.
Still learning Pytorch at the moment, finding it much more intuitive than TF, would love to contribute once I've gained a better understanding.

**Comment by aksub99** [comment]
@EXJUSTICE Thanks for the interest and sorry for the delay. I've been working on refactoring this code this week and just have a couple of bugs to fix before I can push the changes here. The example, final code and README should be ready by tomorrow.

**Comment by EXJUSTICE** [comment]
Nice dude!

No rush at all.
Would love to bug test it and eventually contribute to the repo!



-------- Original Message --------
Subject: Re: [aksub99/MolDQN-pytorch] Hi, would you upload a jupyter notebook about how to use your code? (#1)
From: Akshay Subramanian
To: aksub99/MolDQN-pytorch
CC: "Xu, Yijie" ,Mention


@EXJUSTICE<https://github.com/EXJUSTICE> Thanks for the interest and sorry for the delay. I've been working on refactoring this code this week and just have a couple of bugs to fix before I can push the changes here. The example, final code and README should be ready by tomorrow.

**Comment by aksub99** [comment]
@tbwxmu @EXJUSTICE Sorry once again for the delay. I have updated the README with the installation and running instructions.

@EXJUSTICE I've also created a couple of issues if you're interested to contribute :)

Am closing this issue. 
Addition of Jupyter Notebook example has been made a new issue #6
